pkg/config/config.go

```diff
@@ -48,6 +48,14 @@ type StaticConfig struct {
 	StsScopes            []string `toml:"sts_scopes,omitempty"`
 	CertificateAuthority string   `toml:"certificate_authority,omitempty"`
 	ServerURL            string   `toml:"server_url,omitempty"`
+	// HealthEndpoint is the health check endpoint
+	HealthEndpoint string `toml:"health_endpoint,omitempty"`
```

Comment:
Every new flag complicates the maintenance and user experience. Why do we need to modify all these endpoints?. Why do we need to add a functionality to modify SSE endpoints, as it is already a deprecated functionality?.